test/raft/replication_test.cc

```diff
@@ -305,7 +305,9 @@ RAFT_TEST_CASE(rpc_load_conf_from_log, (test_case{
 // Shrunk later to 2 nodes and then expanded back to 3 nodes.
```

Comment:
> This short term fix disables the packet drop variant of the specified test.
It should be safe to re-enable it once the whole framework is re-worked to
remove these hard coded assumptions.

I think we should open an issue to re-work this test framework. Can you do this? If you have some ideas, obervations, etc., you can mention them in that issue.